assets/scss/components/_toggle-button.scss

```diff
@@ -2,28 +2,73 @@
 Toggle Button
 
 Markup:
-<span class="toggle-btn {{modifier_class}}">Toggle Button</span>
-
-.toggle-btn--on - The toggle button is on
-.toggle-btn--off - The toggle button is off
+<span class="toggle-buttons">
+  <span class="error-notification inline-block toggle__error">Error state</span>
+  <a href="#" class="toggle__button">link</a>
+  <button class="toggle__button">link</button>
+  <span class="toggle--on">on</span>
+  <span class="toggle--off">off</span>
+  <span class="toggle--disabled">disabled</span>
+</span>
 
 Styleguide Toggle Button
 */
 
-
-.toggle-btn {
-  padding: em(5) em(10) em(3) em(10);
+.toggle-buttons {
   display: inline-block;
-  line-height: 1.4em;
+  white-space: nowrap;
+}
+
+.toggle__button {
+  background-color: transparent;
+  box-shadow: none;
+  text-decoration: underline;
+  color: $link-colour;
+  border: none;
+  &:active {
+    top: 0;
+    box-shadow: none;
+  }
+  &:hover,
+  &:focus {
+    outline: none;
+    color: $link-hover-colour;
+    background-color: transparent;
+  }
+  &:disabled,
+  [disabled] {
+    background-color: transparent;
+    &:hover,
+    &:focus {
+      background-color: transparent;
+    }
+  }
+}
+
+.toggle__error {
+  padding: em(5) em(40) em(3) em(10);
 }
 
-.toggle-btn--on {
+.toggle--on {
```

Comment:
`toggle-button` and `toggle-button--active` would be more reusable than `off` and `on`